x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -335,15 +336,29 @@ export class FleetPlugin
     });
     licenseService.start(this.licensing$);
 
-    const fleetServerSetup = startFleetServerSetup();
-
     this.telemetryEventsSender.start(plugins.telemetry, core);
 
+    const logger = appContextService.getLogger();
```

Comment:
@joshdover - I've added some logging throughout the setup process, but because we call `setup` via the API endpoint on the initial page load for Fleet, we get some level of log pollution, e.g.

![Screen Shot 2021-11-09 at 8 12 15 AM](https://user-images.githubusercontent.com/6766512/140930411-c8ba1b57-a295-4f85-b723-85a0b3fc5657.png)

I don't think this is too severe, but it might be confusing to users working with Kibana logs. Do you have any thoughts here?

Reply:
I think it makes sense to have this level of detail at the debug level and just the overall "start", "succeed" and "fail + reason" logs at the info level. Does that make sense?